### ISSUE TITLE
Set a CSS class on body when a popup is displayed

### ISSUE THREAD

**Issue by jpommerening** [LaxarJS/ax-popup-widget#11]
For styling reasons, would it be ok, to add a class to the `body` element when a popup is displayed?

I'm trying to style child elements of the document body differently based on the visibility of a popup. Currently, this is possible if I use an attribute selector, like this:

``` css
body[style*="overflow"] > div.the-div-i-want-to-style {
   /* my css properties */
}
```

Despite the hackyness, due to a [bug](https://bugs.webkit.org/show_bug.cgi?id=64372) in WebKit this does not work consistently for a majority of mobile devices, so it would be nice to have a CSS class to rely on.

**Comment by x1B** [comment]
Good Idea. We should probably name the class in a way that is not specific to the popup-widget though (so that other widgets do not need to know about that).

Bootstrap-CSS actually has a class for this: `modal-open`

So, feel free to submit a PR!

**Comment by jonaswjs** [comment]
Implemented on `master` (`v2.3.0`).
